participants/1-introduction/Cofa12/mahmoud gamal

```diff
@@ -0,0 +1,30 @@
+# mahmoud gamal
```

Comment:
Could you please delete this file? It's not required, we already have the `Readme.md` file for that :)

Reply:
I have done it